dump_views.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+today=$(date +%Y%m%d-%H%M%S)
+
+echo "Started at $today"
+
+mkdir -p csv_export
+
+for table in location admin1 admin2 population operational_presence humanitarian_needs
+do
+  echo "Saving $table"
+  docker exec -t postgres-container psql -U postgres -d hapi -c "\copy (select * from ${table}_view) TO STDOUT (FORMAT csv, DELIMITER ',',  HEADER);" | tee >(gzip > csv_export/${table}.csv.gz) >(md5sum > csv_export/${table}.hash) >/dev/null
+done
+
+docker exec -t postgres-container pg_dump -U postgres -Fc hapi -f hapi_db.pg_restore
+docker cp postgres-container:/hapi_db.pg_restore csv_export/hapi_db.pg_restore
```

Comment:
Should we keep the DB restore file in a separate directory on Github? It may get a bit lost in all of the csvs and hashes

Reply:
Yes I've done that now